cypress/features/regression/carousel.feature

```diff
@@ -1,64 +1,53 @@
     Then slide <index> title is "<title>"
     Examples:
-      | index | title                |
-      | 0     | Slide 1              |
-      | 1     | Full clickable slide |
-      | 2     | Slide 3              |
-      | 3     | Slide 4              |
-      | 4     | Slide 5              |
+      | index | title                | nameOfObject |
+      | 0     | Slide 1              | slideIndex0  |
+      | 1     | Full clickable slide | slideIndex1  |
+      | 2     | Slide 3              | slideIndex2  |
+      | 3     | Slide 4              | slideIndex3  |
+      | 4     | Slide 5              | slideIndex4  |
 
   @positive
   Scenario Outline: Move Carousel left to <index>
-    Given I select slideIndex to "<index>"
+    Given I open default "Carousel" component in noIFrame with "carousel" json from "commonComponents" using "<nameOfObject>" object name
     When I move carousel "left"
     Then slide <indexSlide> title is "<title>"
     Examples:
-      | index | indexSlide | title                |
-      | 1     | 0          | Slide 1              |
-      | 2     | 1          | Full clickable slide |
-      | 3     | 2          | Slide 3              |
-      | 4     | 3          | Slide 4              |
+      | index | indexSlide | title                | nameOfObject |
+      | 1     | 0          | Slide 1              | slideIndex1  |
+      | 2     | 1          | Full clickable slide | slideIndex2  |
+      | 3     | 2          | Slide 3              | slideIndex3  |
+      | 4     | 3          | Slide 4              | slideIndex4  |
 
   @positive
   Scenario Outline: Move Carousel right to <index>
-    Given I select slideIndex to "<index>"
+    Given I open default "Carousel" component in noIFrame with "carousel" json from "commonComponents" using "<nameOfObject>" object name
     When I move carousel "right"
     Then slide <indexSlide> title is "<title>"
     Examples:
-      | index | indexSlide | title                |
-      | 0     | 1          | Full clickable slide |
-      | 1     | 2          | Slide 3              |
-      | 2     | 3          | Slide 4              |
-      | 3     | 4          | Slide 5              |
+      | index | indexSlide | title                | nameOfObject |
+      | 0     | 1          | Full clickable slide | slideIndex0  |
+      | 1     | 2          | Slide 3              | slideIndex1  |
+      | 2     | 3          | Slide 4              | slideIndex2  |
+      | 3     | 4          | Slide 5              | slideIndex3  |
 
   @positive
   Scenario: Carousel previous arrow is disabled
-    When I select slideIndex to "0"
+    When I open default "Carousel" component in noIFrame with "carousel" json from "commonComponents" using "slideIndex0" object name
     Then previous button is disabled
 
   @positive
   Scenario: Carousel next arrow is disabled
-    When I select slideIndex to "4"
-    Then next button is disabled
-
-  @positive
```

Comment:
I think we should leave this TC in this file instead of creating a new one because it's not necessary to have one more file for just one TC